src/list_of_files.cmake

```diff
@@ -65,7 +65,6 @@ set(fortran_src
   blocks.f
   bort.f
   bort2.f
-  bvers.f
```

Comment:
Don't take it out of this list, it is still needed.

Reply:
I don't understand why this is still needed, because the filename is now bvers.f.in, not bvers.f

Reply:
During the cmake step, cmake turns bvers.f.in into bvers.f, so by the time the make happens, it is there.